### PR TITLE
Make `round_integer` work for positive decimal places

### PR DESCRIPTION
**Pull request by GregoryTravis**
### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.
- [x] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

**Comment by AdRiley** [comment]
Do we need some tests for this?

**Comment by GregoryTravis** [comment]
Added just a few tests for `round_integer`. This case was not being tested because the backends' builtin rounding is used for this particular case, but we should run all the tests against the non-builtin methods as well: #11489.

File: test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso [hunk 1]
```diff
@@ -180,6 +180,19 @@ add_column_operation_specs suite_builder setup =
             3.1 . round 16 . should_fail_with Illegal_Argument
             3.1 . round -16 . should_fail_with Illegal_Argument
 
+        group_builder.specify "Non-builtin integer column rounding" <|
+            cases =
+                [[0, 0, 0], [3, 0, 3], [-3, 0, -3], [3, 1, 3], [-3, 1, -3]] +
```
Comment (AdRiley) [inline test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso]: Do + work at the end for multi-line?

Comment (GregoryTravis) [reply test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso]: Ah, I knew that was going to bite me someday. Fixed.

**Review by AdRiley — APPROVED** [review]

**Pull request merged**